tests/libs/estdlib/tests.erl

```diff
@@ -60,6 +60,7 @@ get_non_networking_tests(OTPVersion) when
 get_non_networking_tests(_OTPVersion) ->
     [
         test_apply,
+        test_binary,
```

Comment:
We usually test stuff that is implemented using NIFs in `tests/test.c` (those tests are located in `tests/erlang_tests/`).
So if test_binary contains only tests for NIFs it should be moved elsewhere.